core/src/main/java/com/sequenceiq/cloudbreak/converter/JsonToOpenStackCredentialConverter.java

```diff
@@ -27,7 +27,13 @@ public OpenStackCredential convert(CredentialRequest source) {
         openStackCredential.setUserName(encryptor.encrypt(userName));
         String password = String.valueOf(source.getParameters().get(OpenStackCredentialParam.PASSWORD.getName()));
         openStackCredential.setPassword(encryptor.encrypt(password));
+        openStackCredential.setKeystoneVersion(String.valueOf(source.getParameters().get(OpenStackCredentialParam.KEYSTONE_VERSION.getName())));
```

Comment:
If no KEYSTONE_VERSION is sent in the request, set the keystone version to "cb-keystone-v2" for backward compatibilty, please.